src/components/marginals/index.js

```diff
@@ -1,3 +1,4 @@
-export { default as Footer } from './Footer';
-export { default as Navbar } from './Navbar';
-export { default as NavItems } from './NavItems';
+/* eslint-disable import/no-cycle */
```

Comment:
Could you just once check why this error is being shown? Might have issues going ahead

Reply:
Sure